src/client/executor.rs

```diff
@@ -70,6 +79,13 @@ impl Client {
         op: T,
         session: &mut ClientSession,
     ) -> Result<T::O> {
+        // TODO RUST-9: allow unacknowledged write concerns
```

Comment:
Can we remove the check above for this if we're going to start checking it here, or do we need to do it in both places?

Reply:
removing this method and sending everything through `execute_operation` instead